python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -594,8 +599,6 @@ def update_effective_time_stats(self, summary, exposure):
             Exposure to grab band and exposure time metadata
 
         """
-        self.log.info("Updating effective exposure time")
-
```

Comment:
This should not have been left in during previous commits. No other `update` function logs that it is running.

Reply:
Ah, OK. By "previous commits" you mean previous merges though, because if this was commits on this PR then it wouldn't show in the diff. Fair enough though.

Reply:
Yes, commits on a previous PR and merge.